### PR TITLE
Blood Rage rulebook

### PR DESCRIPTION
**Pull request by andrewmassengale**
Adding Blood Rage rulebook and updating corresponding homepage.yml file.

### CODE REVIEW

File: rulebooks/blood-rage.md [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+title: Blood Rage
+source: https://www.orderofgamers.com/downloads/BloodRage_v1.2.pdf
+tags:
+  - action-point-allowance-system
+  - area-control
+  - card-drafting
+  - hand-management
+---
+
+# SETUP
```
Comment (camden) [inline rulebooks/blood-rage.md]: @andrewmassengale just for consistency with the other rulebooks, can we make this title case?

Comment (camden) [inline rulebooks/blood-rage.md]: @andrewmassengale just for consistency with the other rulebooks, can we make this title case?

Comment (andrewmassengale) [reply rulebooks/blood-rage.md]: This is done.

**Comment by camden** [comment]
Thanks!!

**Pull request merged**